modules/sdk-coin-trx/src/lib/utils.ts

```diff
@@ -13,6 +13,19 @@ import {
   TriggerSmartContract,
 } from './iface';
 import { ContractType, PermissionType } from './enum';
+import { AbiCoder, hexConcat } from 'ethers/lib/utils';
+
+const ADDRESS_PREFIX_REGEX = /^(41)/;
+const ADDRESS_PREFIX = '41';
+
+export const tokenMainnetContractAddresses = [
+  'TEkxiTehnzSmSe2XqrBj4w32RUN966rdz8',
+  'TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t',
+];
+export const tokenTestnetContractAddresses = [
+  'TSdZwNqpHofzP6BsBKGQUWdBeJphLmF6id',
```

Comment:
nit: would maybe make this an object so we can refer to the contracts by their names, like:
tokenMainnetContractAddresses.usdcContract, but also maybe that doesn't make sense for this usecase!

Reply:
i did think about this, but i think for this usecase of just checking if an array includes the token contract address i dont necessarily have to do that. it just requires extra logic otherwise that we dont need